### PR TITLE
Feature/improve readme files

### PR DESCRIPTION
**Pull request by rfelber**
Bugfixed the AppVersion type and added some "on the roadmap" modules (should be visible on the website).

### CODE REVIEW

File: hooks/persistence-defectdojo/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "DefectDojo"
+path: "hooks/persistence-defectdojo"
+category: "hook"
+type: "persistenceProvider"
+state: "roadmap"
+usecase: "Publishes all Scan Findings to elasticsearch (ECK)."
```
Comment (J12934) [inline hooks/persistence-defectdojo/README.md:7]: copy paste error

```suggestion
usecase: "Publishes all Scan Findings to DefectDojo."
```

**Review by nigthknight — APPROVED** [review]

File: hooks/persistence-defectdojo/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+<!-- end -->
+
+## About
+
+DefectDojo is an OpenSource Tools for importing and managing findings of security scanner. The DefectDojo persistence provider can be used to create new Engagements for SecurityTests run via the secureCodeBox and import all findings which were identified.
```
Comment (J12934) [inline hooks/persistence-defectdojo/README.md:14]:
```suggestion
DefectDojo is an open-source tool for importing and managing findings of security scanners. The DefectDojo persistence provider can be used to create new Engagements for SecurityTests run via the secureCodeBox and import all findings which were identified automatically to DefectDojo.
```

File: hooks/persistence-defectdojo/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Tools which are supported by the secureCodeBox and DefectDojo (OWASP ZAP, Arachni & Nmap) this is done by importing the raw scan report into DefectDojo. Findings by other secureCodeBox supported scanners are currently not directly supported by DefectDojo. These findings are imported via a generic finding api of defectDojo, which might cause some loss of information about the findings.
```
Comment (J12934) [inline hooks/persistence-defectdojo/README.md:16]:
```suggestion
Tools which are supported both by the secureCodeBox and DefectDojo (OWASP ZAP & Nmap) this is done by importing the raw scan report into DefectDojo. Findings by other secureCodeBox supported scanners are currently not directly supported by DefectDojo. These findings are imported via a generic finding API of DefectDojo, which might cause some loss of information on the findings.  
```

**Review by J12934 — CHANGES_REQUESTED** [review]

**Comment by rfelber** [comment]
thx @J12934 for your review! Fixed your suggested change in #74

**Pull request merged**